content/management/configuration-api/v3.2/index.mdx

```diff
@@ -2785,6 +2785,238 @@ curl -X POST \
 
 </Section>
 
+# Auto access rules
+
+Auto access rules allow to assign groups to customer session according to their tracking url and geolocation.  Those rules are
+checked on **Login** and **Update Customer Page** and resulting groups stored in customer session.  When that session starts chat with **Start Chat**
+or **Activate Chat** without `chat.access` stored session groups are used as new thread's access.
+
+## Methods
+
+<Section>
+
+<Text>
+
+### Add auto access rules
+
+
+#### Specifics
+
+|                |                                                                               |
+| -------------- | ----------------------------------------------------------------------------- |
+| Method URL     | `https://api.livechatinc.com/v3.2/configuration/action/add_auto_access_rules` |
+| Required scopes| `access_rules:rw`                                                             |
+
+
+| Parameter      | Required     | Data type | Notes                         |
+| -------------- | ------------ | --------- | ----------------------------- |
+| `access`__*__  | Yes          | `object`  | Destination access            |
+| `rules`__**__  | Yes          | `object`  | Rules to check for scope auto |
+| `description`  | No           | `string`  | Auto chat scopes description  |
+
+__*__ `access` object:
+    * `groups`: `[1, 2]`
+
+__**__ `rules` possible rules:
+  * `customer_url.<string_filter_type>` - matches on `customer_page.url` from **Login** or `url` from **Update Customer Page**
+    * `<string_filter_type>` possible values (only one is allowed for single customer_url):
+      * `values` (array of `match_object`)
+      * `exclude_values` (array of `match_object`)
+    * `<match_object>` structure:
```

Comment:
`<match_object>` is repeated multiple times here, maybe worth extracting its structure to a separate top-level description (a sibling of `access` & `rules`)? It would make this section shorter and thus easier to grok